content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -578,3 +580,35 @@ Type Checking has the following limitation:
 - Type Checking does not affect the policy behavior in any way. Even if the type checking detects errors, the policy will continue
   to evaluate. If errors do occur during evaluate, the failure policy will decide its outcome.
 - Type Checking does not apply to CRDs, including matched CRD types and reference of paramKind. The support for CRDs will come in future release.
+
+### Variable Composition
```

Comment:
```suggestion
### Variable composition
```

We use sentence case for headings (and, for beta and GA APIs, we like the documentation to match our style guide)